app/agents_v2/primary_agent/reasoning/reasoning_engine.py

```
@@ -13,31 +13,27 @@
 import asyncio
 import re
 
-from langchain_core.messages import BaseMessage, HumanMessage
+from langchain_core.messages import BaseMessage, HumanMessage, SystemMessage
```

Ruff (0.12.2)

`langchain_core.messages.BaseMessage` imported but unused

Remove unused import: `langchain_core.messages.BaseMessage`

(F401)